include/WRAPPER_SLURM

```diff
@@ -1,28 +1,47 @@
 #!/bin/bash
+#SBATCH --job-name="lcdb-wf"
+#SBATCH --partition="niddk,norm"
+#SBATCH --time=24:00:00
 
-if [ -z $SNAKEFILE ]; then
-    SNAKEFILE=$1
-fi
```

Comment:
did not find this to be needed, we can add it back but `$1` should always be the snakefile.